server/src/models.js

```diff
@@ -122,6 +123,7 @@ export const defineModels = (db) => {
   // event responses belong to a combination of user/event
   EventResponseModel.belongsTo(EventModel);
   EventModel.hasMany(EventResponseModel);
+  EventModel.hasOne(EventLocationModel, { as: 'location' });
```

Comment:
maybe call this `primaryLocation` or something to disambiguate it from `locations` which is the collection of all locations relevant to the event ("lhq", "scene" are the 2 we would have most of the time but there might be others like "staging area", "locked gate" we could add the ability to add or something.